Sentinel/Classes/Core/Internal/Image+Assets.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  Image+Assets.swift
+//  Sentinel
+//
+//  Created by Zvonimir Medak on 10.12.2024..
+//
+
+import SwiftUI
+
+// There's an issue with the generated assets, and local cocoapods which should be resolved in Xcode 16.1
+extension Image {
+    enum SentinelImages{}
+}
+
+extension Image.SentinelImages {
+    static var device = Image.load(using: "device")
+    static var application = Image.load(using: "application")
+    static var tools = Image.load(using: "tools")
+    static var preferences = Image.load(using: "preferences")
+    static var performance = Image.load(using: "performance")
+}
+
+extension Image {
+
+    static func load(using name: String) -> Image {
```

Comment:
I would rather make the return type as `Optional` and create additional method that will contain `defaultImage` parameter (which can be what you are returning here - it can also be defined as a default parameter) and that would resolve potential unexpected behaviour.

P.S. Also add comments ono both methods.